### PR TITLE
Removal of personal data from mod_mam

### PR DESCRIPTION
**Pull request by zofpolkowska**
Removed are only pm conversations from archive.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[6642.6](https://travis-ci.org/esl/MongooseIM/jobs/539737895) / Erlang 19.3 / elasticsearch_and_cassandra_mnesia / 6f917dc37acc55cb087f70896913b03e178c5b4a
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3)/ [big](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/index.html)
**OK: 495** / **Failed: 57** / **User-skipped: 47** / **Auto-skipped: 126** 
<details><summary>mam_SUITE:muc03:muc_show_x_user_to_moderators_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_to_moderators_in_anon_rooms.354.html)
</details>
<details><summary>mam_SUITE:muc03:muc_delete_x_user_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_delete_x_user_in_anon_rooms.342.html)
</details>
<details><summary>mam_SUITE:muc03:muc_archive_request</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_archive_request.367.html)
</details>
<details><summary>mam_SUITE:muc03:muc_show_x_user_for_your_own_messages_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_for_your_own_messages_in_anon_rooms.358.html)
</details>
<details><summary>mam_SUITE:muc03:muc_multiple_devices</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_multiple_devices.371.html)
</details>
<details><summary>mam_SUITE:muc04:muc_delete_x_user_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_delete_x_user_in_anon_rooms.398.html)
</details>
<details><summary>mam_SUITE:muc04:muc_archive_request</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_archive_request.386.html)
</details>
<details><summary>mam_SUITE:muc04:muc_show_x_user_to_moderators_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_to_moderators_in_anon_rooms.402.html)
</details>
<details><summary>mam_SUITE:muc04:muc_show_x_user_for_your_own_messages_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_for_your_own_messages_in_anon_rooms.406.html)
</details>
<details><summary>mam_SUITE:muc04:muc_multiple_devices</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_multiple_devices.375.html)
</details>
<details><summary>mam_SUITE:muc06:muc_show_x_user_to_moderators_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_to_moderators_in_anon_rooms.438.html)
</details>
<details><summary>mam_SUITE:muc06:muc_delete_x_user_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_delete_x_user_in_anon_rooms.434.html)
</details>
<details><summary>mam_SUITE:muc06:muc_archive_request</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_archive_request.450.html)
</details>
<details><summary>mam_SUITE:muc06:muc_show_x_user_for_your_own_messages_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_for_your_own_messages_in_anon_rooms.442.html)
</details>
<details><summary>mam_SUITE:muc06:muc_multiple_devices</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_multiple_devices.458.html)
</details>
<details><summary>mam_SUITE:muc_rsm_all:init_per_group</summary>


```erlang
{'EXIT',{test_case_failed,"Respond size is 0, 15 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.init_per_group.403.html)
</details>
<details><summary>mam_SUITE:muc03:muc_archive_request</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_archive_request.470.html)
</details>
<details><summary>mam_SUITE:muc03:muc_show_x_user_to_moderators_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_to_moderators_in_anon_rooms.435.html)
</details>
<details><summary>mam_SUITE:muc03:muc_show_x_user_for_your_own_messages_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_for_your_own_messages_in_anon_rooms.439.html)
</details>
<details><summary>mam_SUITE:muc03:muc_delete_x_user_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_delete_x_user_in_anon_rooms.431.html)
</details>
<details><summary>mam_SUITE:muc03:muc_multiple_devices</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_multiple_devices.478.html)
</details>
<details><summary>mam_SUITE:muc04:muc_delete_x_user_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_delete_x_user_in_anon_rooms.475.html)
</details>
<details><summary>mam_SUITE:muc04:muc_archive_request</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_archive_request.459.html)
</details>
<details><summary>mam_SUITE:muc04:muc_show_x_user_to_moderators_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_to_moderators_in_anon_rooms.479.html)
</details>
<details><summary>mam_SUITE:muc04:muc_show_x_user_for_your_own_messages_in_anon_rooms</summary>


```erlang
{error,{test_case_failed,"Respond size is 0, 1 is expected."}}
```

[Report log](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6642/elasticsearch_and_cassandra_mnesia.19.3/big/ct_run.test%40travis-job-9cccbbd4-8a47-4c93-98b5-2d77f9b033a1.2019-05-31_14.45.37/big_tests.tests.mam_SUITE.logs/run.2019-05-31_14.45.41/mam_suite.muc_show_x_user_for_your_own_messages_in_anon_rooms.494.html)
</details>

35 errors were truncated

**Comment by mongoose-im** [comment]
[6643.1](https://travis-ci.org/esl/MongooseIM/jobs/539766441) / Erlang 19.3 / small_tests / db133de7d0ca6c3fed0b22fc64234a3f8ce0f135
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6643/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6643/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-31_15.42.38/lib.mongooseim.logs/run.2019-05-31_15.42.39/suite.log.html)

**Comment by mongoose-im** [comment]
[6649.1](https://travis-ci.org/esl/MongooseIM/jobs/540810248) / Erlang 19.3 / small_tests / 55ffa0e26c81c7a643a995ca4d85e29a955d6f70
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6649/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6649/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-06-03_15.33.58/lib.mongooseim.logs/run.2019-06-03_15.33.59/suite.log.html)

**Comment by mongoose-im** [comment]
[6679.1](https://travis-ci.org/esl/MongooseIM/jobs/544732752) / Erlang 20.3 / small_tests / c6293b9cd09b7b2b90affec1d05ba717a67d45e2
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6679/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6679/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-12_13.12.58/lib.mongooseim.logs/run.2019-06-12_13.12.59/suite.log.html)

File: src/mam/mod_mam.erl [hunk 1]
```diff
@@ -336,12 +337,47 @@ filter_packet({From, To=#jid{luser=LUser, lserver=LServer}, Acc, Packet}) ->
...
+            E:R ->
+                Stack = erlang:get_stacktrace(),
+                ?WARNING_MSG("issue=remove_user_failed "
+                "reason=~p:~p "
+                "stacktrace=~1000p ", [E, R, Stack]),
+                ok
+        end end, [mod_mam_cache_user | Backends]).
+
+try_archive_id_int(Host, #jid{ luser = LUser } = _ArcJID) ->
+    UserIDBackends = mongoose_lib:find_behaviour_implementations(ejabberd_gen_mam_user),
```
[the remark this answers is not in the document]
Comment (fenek) [reply src/mam/mod_mam.erl]: I think that the latter option makes more sense.

File: src/mam/mod_mam_cassandra_arch.erl [hunk 2]
```diff
@@ -201,12 +202,16 @@ remove_archive_offsets_query_cql() ->
...
     "SELECT DISTINCT user_jid, with_jid FROM mam_message WHERE user_jid = ?".
 
-remove_archive(Acc, _Host, _UserID, UserJID) ->
+remove_archive(Acc, Host, UserID, UserJID) ->
+    remove_archive(Host, UserID, UserJID),
+    Acc.
+
+remove_archive(Host, _UserID, UserJID) ->
+    ensure_params_loaded(Host),
+    PoolName = mod_mam_cassandra_arch_params:pool_name(),
```
[the remark this answers is not in the document]
Comment (fenek) [reply src/mam/mod_mam_cassandra_arch.erl]: It's not really a subject of this PR to improve consistency of `pool_name` behaviour.

File: src/mam/mod_mam_rdbms_user.erl [hunk 3]
```diff
@@ -10,11 +10,14 @@
 %%%-------------------------------------------------------------------
 -module(mod_mam_rdbms_user).
 
+-behaviour(ejabberd_gen_mam_user).
```
[the remark this answers is not in the document]
Comment (fenek) [reply src/mam/mod_mam_rdbms_user.erl]: Do we care about this ID after user is removed? It's used only for retrieval, correct?

File: src/mam/mod_mam_cassandra_arch.erl (hunk 2 above)
[the remark this answers is not in the document]
Comment (fenek) [reply src/mam/mod_mam_cassandra_arch.erl]: Aaaaand I decided to remove it after all. :D

[the remark this answers is not in the document]
Comment (fenek) [reply src/mam/mod_mam_cassandra_arch.erl]: Actually I misunderstood your comment at first. :P

File: src/mam/mod_mam_rdbms_user.erl (hunk 3 above)
[the remark this answers is not in the document]
Comment (DenysGonchar) [reply src/mam/mod_mam_rdbms_user.erl]: yes, right now it's required only for mam muc gdpr retrieval. but i would prefer to keep data in RDBMS consistent.

**Comment by mongoose-im** [comment]
[6693.1](https://travis-ci.org/esl/MongooseIM/jobs/547138243) / Erlang 20.3 / small_tests / da31b5b0a91196a8de3074c81466f6e877b614e2
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6693/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6693/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-18_09.22.42/lib.mongooseim.logs/run.2019-06-18_09.22.43/suite.log.html)

**Comment by mongoose-im** [comment]
[6695.1](https://travis-ci.org/esl/MongooseIM/jobs/547402713) / Erlang 20.3 / small_tests / dcda2bb4bf80225fae8f438288d9e1d896d0741f
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6695/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6695/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-18_20.12.44/lib.mongooseim.logs/run.2019-06-18_20.12.45/suite.log.html)

**Comment by mongoose-im** [comment]
[6704.1](https://travis-ci.org/esl/MongooseIM/jobs/547864670) / Erlang 20.3 / small_tests / 07f50d9f4c85fa5f8dbb2400148e3b627e17c705
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6704/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6704/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-19_18.57.16/lib.mongooseim.logs/run.2019-06-19_18.57.17/suite.log.html)

**Comment by mongoose-im** [comment]
[6705.1](https://travis-ci.org/esl/MongooseIM/jobs/548605855) / Erlang 20.3 / small_tests / df5789cf93621814e8139638ef28322962a39166
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6705/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6705/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-21_09.59.27/lib.mongooseim.logs/run.2019-06-21_09.59.28/suite.log.html)

File: src/mam/mod_mam_rdbms_user.erl [hunk 4]
```diff
@@ -15,6 +15,7 @@
 
 %% ejabberd handlers
 -export([archive_id/3,
+         remove_archive/3,
```
Comment (DenysGonchar) [inline src/mam/mod_mam_rdbms_user.erl]: no need to export this one

**Comment by mongoose-im** [comment]
[6715.1](https://travis-ci.org/esl/MongooseIM/jobs/549816122) / Erlang 20.3 / small_tests / 48a91aae5059d8eefcd3c6aceb8b65f8b6c3f88c
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2328/6715/small_tests.20.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2328/6715/small_tests.20.3/small/ct_run.mongooseim@localhost.2019-06-24_17.31.37/lib.mongooseim.logs/run.2019-06-24_17.31.38/suite.log.html)

**Pull request merged**